plugins/http_plugin/tests/unit_tests.cpp

```diff
@@ -694,12 +704,19 @@ BOOST_FIXTURE_TEST_CASE(requests_in_flight, http_plugin_test_fixture) {
       return count_of_status_replies;
    };
 
+   auto wait_for_no_bytes_in_flight = [&](uint16_t max = std::numeric_limits<uint16_t>::max()) {
+      while (http_plugin->bytes_in_flight() > 0 && --max)
+         std::this_thread::sleep_for(std::chrono::milliseconds(5));
+      BOOST_CHECK(max > 0);
+   };
+
 
    //8 requests to start with
    send_requests(8u);
    std::unordered_map<boost::beast::http::status, size_t> r = scan_http_replies();
    BOOST_REQUIRE_EQUAL(r[boost::beast::http::status::ok], 8u);
    connections.clear();
+   wait_for_no_bytes_in_flight();
```

Comment:
this test case has to do with requests in flight, not bytes, so this feels mismatched but maybe it still serves enough of its purpose?

Reply:
A requests in flight would be better and useful to verify in both cases. I'll add it.